pkgs/applications/graphics/openscad/default.nix

```diff
@@ -64,6 +64,11 @@ mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace src/FileModule.cc \
+      --replace-fail 'fs::is_regular' 'fs::is_regular_file'
```

Comment:
this seems like it might cause an issue if the file Is changed to use `fs::is_regular_file`. maybe sed is better as can specify word boundaries.

```
sed -i 's/\<fs::is_regular\>/&_file/' src/FileModule.cc
```

Reply:
How out-of-date is our OpenSCAD version? It seems upstream fixed this issue in 2022: https://github.com/openscad/openscad/commit/cc51ea0a0dd14075df0f9af09ed4e20cce7fcb4c

Reply:
Ugh, upstream haven't made a release after 2021...

Reply:
> this seems like it might cause an issue if the file Is changed to use fs::is_regular_file

Yes, but (1) it'll still cause a build time failure and (2) upstream already fixed this so on the next update using --replace-fail will fail at build time (good) unlike sed (bad).

Reply:
> Ugh, upstream haven't made a release after 2021...

For those who want to track progress on the next release: https://github.com/orgs/openscad/projects/2

Reply:
> > this seems like it might cause an issue if the file Is changed to use fs::is_regular_file
> 
> Yes, but (1) it'll still cause a build time failure and (2) upstream already fixed this so on the next update using --replace-fail will fail at build time (good) unlike sed (bad).

I guess we can agree to disagree on this one.